crates/re_data_ui/src/image.rs

```diff
@@ -109,26 +109,27 @@ fn tensor_ui(
         UiVerbosity::Small => {
             ui.horizontal_centered(|ui| {
                 if let Some(texture) = &texture_result {
-                    let max_height = 24.0;
-                    let max_size = Vec2::new(4.0 * max_height, max_height);
-                    show_image_at_max_size(
+                    let preview_size = Vec2::splat(24.0);
+                    show_image_preview(
                         ctx.render_ctx,
                         ctx.re_ui,
                         ui,
                         texture.clone(),
                         &debug_name,
-                        max_size,
+                        preview_size,
+                        false,
                     )
                     .on_hover_ui(|ui| {
-                        // Show larger image on hover
-                        let max_size = Vec2::splat(400.0);
-                        show_image_at_max_size(
+                        // Show larger image on hover.
+                        let preview_size = Vec2::splat(400.0);
```

Comment:
Will small 20x20 images now be blown up to 400x400?

Reply:
they would with this. But I'd reason this is actually a good thing. Should a 2x2 image really show as a 2x2 preview?

Reply:
it is nearest neighbor already :)
As discussed on slack as well I left a note ✔️